tests/behavior/replacementkeeper_test.lua

```diff
@@ -67,6 +67,25 @@ function on_cycle()
         set_robot_pose(1, { -3.2, 3 }, -1.5707963267948966);
     end
 
+    if state.cycle_count == 2150 then
+        unpenalize(1);
+        state.game_controller_state.penalties.one = nil;
+    end
+
+    if state.cycle_count == 2600 then
+        state.ball = {
+            position = { 0.0, 0.0 },
+            velocity = { 0.0, 0.0 },
+        }
+    end
+
+    if state.cycle_count == 3000 then
+        state.ball = {
+            position = { -4.0, 0.0 },
+            velocity = { 0.0, 0.0 },
+        }
+    end
```

Comment:
Why did you add these ball position teleports?

Reply:
In order to check if the replacmentkeeper can still become striker and is not forced to stay replacmentkeeper until the keeper is back in the goal.

Reply:
What do you think about 
```lua

    if state.cycle_count == 2150 then
        unpenalize(1);
        state.game_controller_state.penalties.one = nil;
    end

    if state.cycle_count == 3550 then
        state.ball = {
            position = { 0.0, 0.0 },
            velocity = { 0.0, 0.0 },
        }
    end

    if state.cycle_count == 3600 then
        penalize(1);
        state.game_controller_state.penalties.one = {
            Manual = {
                remaining = {
                    nanos = 0,
                    secs = 5
                },
            }
        };
        set_robot_pose(1, { -3.2, 3 }, -1.5707963267948966);
    end

    if state.cycle_count == 3900 then
        unpenalize(1);
        state.game_controller_state.penalties.one = nil;
    end

    if state.cycle_count == 4650 then
        state.ball = {
            position = { -4.0, 0.0 },
            velocity = { 0.0, 0.0 },
        }
    end
```

As this would test both situations within the same file by penalizing the striker twice?

Reply:
This indeed looks like a better version, therefore I changed the file to this.